site/en/blog/fledge-api/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: 'layouts/blog-post.njk'
-title: FLEDGE API developer guide
+title: Protected Audience API developer guide
```

Comment:
Leave this blog as FLEDGE and add the outdated header to it. As the content is from Jan '22 I think it will be technically out of date anyway.

Reply:
done